src/assets/googlr-form-script.gs

```diff
@@ -0,0 +1,131 @@
+/**
+ * Array of Discord webhook URLs where the form responses will be sent.
+ */
+const webhooks = [
+  'https://discord.com/api/webhooks/1241812632013639750/cG-rCFu2bO58d3eIiLwisCYjpQYA26qxK6-Sl2tUk-s2PJh7Q9utYdJ_FgR6f8FjfJ-j',
```

Comment:
use an environment variable, links like these shouldn't be commited in the source code

Reply:
fix the type in the file name googlr -> google